assets/activestate.yaml.python.tpl

```diff
@@ -2,6 +2,7 @@ scripts:
   - name: activationMessage
     language: {{.Language}}
     value: |
+      # -*- coding: utf-8 -*-
```

Comment:
This needs to be here for python2.  It otherwise barks when we try to print non-ascii characters, and doesn't print the activation message.